include/int128.h

```diff
@@ -29,7 +29,8 @@ extern "C" {
 #include <stdbool.h>
 #include <stdint.h>
 
-#if __GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ > 8)
+#if ((__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ > 8)) && \
+  !defined(__aarch64__)
```

Comment:
Is __int128 not supported or why is this disabled for aarch64?

Reply:
Performs better when aligned on 4 instead of 8. Also, spurious SEGV's observed when using the native __int128 on AArch64 with GCC.